### ISSUE TITLE
addClass('easyPieChart') missing from new versions

### ISSUE THREAD

**Issue by helgatheviking** [rendro/easy-pie-chart#149]
On the [demo page](http://rendro.github.io/easy-pie-chart/)  the class `easyPieChart` is added to the element that the jquery function is called on. However if I look at the demo page's source it would appear that the version of EasyPieChart.js is version 1.2.3. The version in the repo is 2.1.7 and there's no longer and `addClass()` which makes styling the element a lot more difficult. 

http://rendro.github.io/easy-pie-chart/javascripts/jquery.easy-pie-chart.js
versus
https://github.com/rendro/easy-pie-chart/blob/master/dist/easypiechart.js

Could you add the `addClass()` back to the `init()` method? 

```
var init = function() {
    this.el = el;
    this.options = options;
    $(el).addClass('easyPieChart');
    ....
    }
```

**Comment by helgatheviking** [comment]
It looks like calculated CSS styles are missing too.... such as lineheight, etc, which also contribute to recreating the styles in your demo (specifically the percentage in the middle of the chart)

```
        // merge user options into default options
        for (var i in defaultOptions) {
            if (defaultOptions.hasOwnProperty(i)) {
                options[i] = opts && typeof(opts[i]) !== 'undefined' ? opts[i] : defaultOptions[i];
                if (typeof(options[i]) === 'function') {
                    options[i] = options[i].bind(this);
                }
            }
        }

        $(el).addClass('easyPieChart').css({
            width: options.size,
            height: options.size,
            lineHeight: "" + options.size + "px"
        }); 
```

**Comment by gentunian** [comment]
Yep. I can't make this to work. Only works on codepen [here](http://codepen.io/rendro/pen/vrezp). If I move the exactly generated CSS and HTML into my code with the latest version will not work.

Not sure why, I'm still working with it. As a start `easyChartPie` class is not applying. Anyway, adding the class manually in the inspector won't work either...

**Comment by gentunian** [comment]
Ok, this is what I did. I've downloaded this link:

http://rendro.github.io/easy-pie-chart/javascripts/jquery.easy-pie-chart.js

As @helgatheviking said, will work OK. I think this is related to this [bug](https://github.com/rendro/easy-pie-chart/issues/93)

**Comment by rendro** [comment]
Closing as a usage question rather than a defect — sorry it went unanswered.

Most of these come down to the same two things, both now documented in the 3.0 Readme:

- **Text in the middle of the chart** is your own element, positioned with CSS. The library only draws the canvas. `examples/index.html` in the repo shows the markup and CSS, including live percentage labels.
- **Driving the value** (from scroll, a click, an accordion, an AJAX call, a countdown) is done by calling `chart.update(value)` yourself. The `onStep(from, to, value)` callback is where you'd format a label — prefix, suffix, thousand separators, rounding.

If something here is actually a library defect, please reopen with a reproduction on 3.0.

https://github.com/rendro/easy-pie-chart/releases/tag/v3.0.0
